_posts/2021-05-13-automate-static-site-builds-with-sensenet-webhooks.md

```diff
@@ -0,0 +1,49 @@
+---
+title: "Automate static site builds with sensenet webhooks"
+author: [herflis]
+image: "../img/posts/automate-site-builds-wall-e.jpg"
+tags: [webhooks, integration, build, deployment, netlify, gatsby, travis ci]
+redirect_to: https://www.sensenet.com/blog/2021-05-13-automate-static-site-builds-with-sensenet-webhooks
+---
+
+Integration is a key aspect of digital projects, since preferably, all services used in a project should be connected and work together seamlessly. With sensenet webhooks and the built-in webhook templates it's super easy to achieve such goals. Let's see how!
+
+---
+
+## What is a webhook
+
+In short, using a webhook is a great way for your application to receive events or notifications from a service.
+
+In terms of sensenet it means your repository can send notifications if something happened with a content. You can configure these webhook subscriptions as you wish, you can set what type of content, which subtree, and what specific events to work with and of course you can set the scope of the data (payload) to be sent.
+
+Webhooks can not only notify your app about what happening in your content repository, but also builds up connection between sensenet and your favorite external services. They can be used in many very different cases from sharing information in social media about content changes till rebuilding static websites automatically.
+
+## Triggering static site builds
+
+Once you've setup a static site that pulls in your content during the build process, you're ready to configure webhooks that will be triggered when you are working with content in your repository. The next example teach you how to setup webhooks to trigger builds and deployments of static sites on Netlify, but this is just an example. sensenet of course provides you various other options for Heroku fans, Travis CI users and Gatsby Cloud subscribers among others. Check currently available templates [here](https://docs.sensenet.com/integrations/webhook). Webhook templates helps you create webhooks subscription for specific external services with just a few steps, by filling only a few required fields.
+
+## Configuring a Netlify build hook
+
+First, go to your static site on Netlify and create a build hook for sensenet (it can be found under Site > Site settings > Build & Deploy > Build hooks). This will be url that builds up the connection between your repository and the static site.
```

Comment:
```suggestion
First, go to your static site on Netlify and create a build hook for sensenet (it can be found under Site > Site settings > Build & Deploy > Build hooks). This will be the url that builds up the connection between your repository and the static site.
```